src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_COMPANIES;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+
+/**
+ * Deletes a company identified using it's displayed index from the address book.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Views the company identified by the index number used in the displayed company list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_COMPANY_SUCCESS = "Viewing Company: %1$s";
+
+    private final Index targetIndex;
+
+    /**
+     * Creates an ViewCommand to view the specified {@code Company}
+     * @param targetIndex index of the company in the filtered company list to view
```

Comment:
I think there should be a space between description and @params. Missing full stops too.

Reply:
According to the [Javadocs standards](https://se-education.org/guides/conventions/java/intermediate.html#comments), there should be punctuations behind each parameter description.